README.rst

```diff
@@ -2,7 +2,7 @@
 PySlurm: Slurm Interface for Python
 ***********************************
 
-.. image:: https://travis-ci.org/PySlurm/pyslurm.svg?branch=19.05.0
+.. image:: https://travis-ci.org/PySlurm/pyslurm.svg?branch=master
```

Comment:
You may change this to 20.02.0. I will create a release with this version after this PR is merged.

Reply:
Got it, I changed it.